src/containers/ArticlePage/components/TaxonomyConnectionErrors.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import styled from '@emotion/styled';
+import { AlertCircle } from '@ndla/icons/editor';
+import { colors, spacing } from '@ndla/core';
+import { FieldHeader } from '@ndla/forms';
+import Tooltip from '@ndla/tooltip';
+import { useTranslation } from 'react-i18next';
+import { Resource, Topic } from '../../../modules/taxonomy/taxonomyApiInterfaces';
+
+const StyledWarnIcon = styled(AlertCircle)`
+  height: ${spacing.nsmall};
+  width: ${spacing.nsmall};
+  fill: ${colors.support.red};
+`;
+
+const TaxonomyInfoDiv = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+`;
+
+const StyledId = styled.span`
+  font-style: ${(props: { isVisible: boolean }) => !props.isVisible && 'italic'};
+  color: ${(props: { isVisible: boolean }) =>
+    !props.isVisible ? colors.brand.grey : colors.brand.primary};
```

Comment:
Det at id på feilkobla element vises blått gjør at det ser ut som om det er ei lenke, uten at det er det. Enten bør fargen være svart, eller endå bedre, det kan være ei lenke til plasseringa i strukturen der elementet er brukt. Det gjør det kanskje lettere å rydde opp i etterkant. Kanskje det skulle være en (i) ikon der det står at artikkeltypen kan endres i databasen for å fikse feilen.